LICENSE

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I think having this blank line here is rather standard? Anyway, changing the LICENSE file (even for whitespace) seems like a change that needs a separate PR, so I think we should revert this file here.

Reply:
reverted